<launch>
  <node name="cmd_vel_to_serial" pkg="hovercar_driver" type="hovercar_driver_node" output="screen">
    <param name="port" value="/dev/ttyACM0" />
    <param name="wheel_base" value="0.6" />
    <param name="max_steer_angle" value="30.0" />
  </node>
</launch>
